contracts/periphery/BeraCrocLpErc20.sol

```diff
@@ -45,4 +35,18 @@ contract BeraCrocLpErc20 is ICrocLpConduit, BGTEligibleERC20 {
         _burn(sender, seeds);
         return true;
     }
+
+        // called once by the factory at time of deployment
+    function initialize(address _base, address _quote, uint256 _idx) external {
+        // CrocSwap protocol uses 0x0 for native ETH, so it's possible that base
+        // token could be 0x0, which means the pair is against native ETH. quote
+        // will never be 0x0 because native ETH will always be the base side of
+        // the pair.
+        require(_quote != address(0) && _base != _quote && _quote > _base, "Invalid Token Pair");
```

Comment:
```suggestion
        require(_base != address(0) && _quote > _base, "Invalid Token Pair");
```

Reply:
similar to @po-bera 's suggestion. simply checking quote > base should be sufficient

Reply:
`quote == 1` and `base == 0` would pass.

Reply:
We still want `base == 0` for `BERA`?

Reply:
I see.